frontend/packages/console-shared/src/selectors/common.ts

```diff
@@ -1,6 +1,9 @@
-import * as _ from 'lodash';
+import get from 'ts-get';
 
 import { K8sResourceKind } from '@console/internal/module/k8s';
 
-export const getName = (value: K8sResourceKind): string => _.get(value, 'metadata.name');
-export const getNamespace = (value: K8sResourceKind): string => _.get(value, 'metadata.namespace');
+export const getName = (value: K8sResourceKind) => get(value, (v) => v.metadata.name);
+export const getNamespace = (value: K8sResourceKind) => get(value, (v) => v.metadata.namespace);
+export const getDeletetionTimestamp = (value: K8sResourceKind) =>
+  get(value, (v) => v.metadata.deletionTimestamp);
+export const getUid = (value: K8sResourceKind) => get(value, (v) => v.metadata.uid);
```

Comment:
Could just as easily make the code explicit:
```suggestion
export const getUid = (value: K8sResourceKind) => value.metadata ? value.metadata.uid : undefined;
```

Reply:
We'd want to check for `value` being `undefined` as well

Reply:
Sorry I'm use to working in projects with strict tsconfig options enabled: `strictNullChecks = true`. I saw `value` and not `value?` and assumed it cannot be `null|undefined`. We should look at tightening the tsconfig settings.

```suggestion
export const getUid = (value: K8sResourceKind) => value && value.metadata && value.metadata.uid || undefined;
```